<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>GroupBoxBinLoadDrop</class>
    <widget class="QWidget" name="GroupBoxBinLoadDrop">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>460</width>
                <height>100</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>460</width>
                <height>100</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
                <widget class="QGroupBox" name="groupBox">
                    <property name="minimumSize">
                        <size>
                            <width>0</width>
                            <height>0</height>
                        </size>
                    </property>
                    <property name="title">
                        <string>Bin Load Drop</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_2">
                        <item row="0" column="0">
                            <widget class="QLabel" name="labelBinLoadDrop">
                                <property name="text">
                                    <string>Bin load drop:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="1">
                            <widget class="QPushButton" name="pushButtonBinLoadRelay">
                                <property name="text">
                                    <string>LOAD</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="2">
                            <widget class="QPushButton" name="pushButtonBinDropRelay">
                                <property name="text">
                                    <string>DROP</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="3">
                            <widget class="QLabel" name="labelProximityBinDropStatus">
                                <property name="text">
                                    <string>Status</string>
                                </property>
                                <property name="alignment">
                                    <set>Qt::AlignCenter</set>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
